docs/index.rst

```diff
@@ -1,25 +1,28 @@
-.. currentmodule:: coremltools.converters
-.. autosummary::
-  :nosignatures:
-  :toctree: Converters/generated
+.. coremltools API Reference documentation master file, created by
+   sphinx-quickstart on Mon Apr 19 12:32:37 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
 
-    convert
-    keras.convert
-    caffe.convert
-    onnx.convert
-    libsvm.convert
-    sklearn.convert
-    xgboost.convert
+coremltools API v4.1
```

Comment:
We really want to avoid hardcoding version numbers. Just change this to `coremltools API`.